gateway/1.4/modules/ROOT/pages/flex-architecture-shared-storage.adoc

```diff
@@ -0,0 +1,17 @@
+= Shared Storage Models
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+When configuring shared storage, MuleSoft recommends placing the Redis shared storage service as close to Flex Gateway as possible. 
+
+
+To configure the connection between Flex Gateway and the shared storage service, see:
+* xref:flex-conn-shared-storage.adoc[Configure Shared Storage in Connected Mode]
+* xref:flex-local-shared-storage.adoc[Configure Shared Storage in Local Mode]
+
+
+You can place the Redis shared storage within the same Kubernetes cluster as Flex Gateway or as a public cloud service instantiated within the same Virtual Private Cloud (VPC) as Flex Gateway:
```

Comment:
The second part of this is confusing. Does the shared service get deployed as a public cloud service? If so: ...or you can deploy it as...